listenbrainz/spotify_updater/spotify_read_listens.py

```diff
@@ -234,6 +256,7 @@ def process_all_spotify_users():
                 success=False,
                 error_message=str(e),
             )
+            notify_error(u.musicbrainz_row_id, str(e))
```

Comment:
To confirm, this means that we'll send an email to a user a maximum of one time? (because we'll also set `success=False`, so we won't process this user again). I want to make sure that we don't send a user an email every 10 minutes.

How will a user "reset" their account to reprocess it? Will we remove the last-updated row? or will something set `success`?

Reply:
Yes, if `success == False` then the user isn't processed again, there are tests for this.

Right now, the user would just unlink their account and relink again. There is probably a better flow for this though.